src/org/zaproxy/zap/extension/fuzz/payloads/generator/RegexPayloadGenerator.java

```diff
@@ -57,19 +57,27 @@
 
     private final int numberOfPayloads;
 
+    private boolean isRandomOrder;
+
     public RegexPayloadGenerator(String regex) {
         this(regex, 0);
     }
 
     public RegexPayloadGenerator(String regex, int maxPayloads) {
-        this(regex, maxPayloads, maxPayloads);
+        this(regex, maxPayloads, maxPayloads, false);
+    }
+
+    public RegexPayloadGenerator(String regex, int maxPayloads, boolean isRandomOrder) {
+        this(regex, maxPayloads, maxPayloads, isRandomOrder);
     }
 
-    public RegexPayloadGenerator(String regex, int maxPayloads, int limitCalculationPayloads) {
+    public RegexPayloadGenerator(String regex, int maxPayloads, int limitCalculationPayloads, boolean isRandomOrder) {
+
```

Comment:
Could be removed.

Reply:
no, because it is used  in `RegexPayloadGeneratorUI`

Reply:
I'm referring to the empty line ;)